pkgs/http/lib/src/browser_client.dart

```diff
@@ -45,6 +44,22 @@ BaseClient createClient() {
 class BrowserClient extends BaseClient {
   final _abortController = AbortController();
 
+  final String? _cacheMode;
+
+  /// Create a new browser-based HTTP Client.
+  ///
+  /// If [cacheMode] is provided then it can be used to cache the request
+  /// in the browser.
+  ///
+  /// For example:
+  /// ```dart
+  /// final client = BrowserClient(cacheMode: 'reload');
+  /// ```
+  ///
+  /// Defaults to `default`.
+  /// https://developer.mozilla.org/en-US/docs/Web/API/Request/cache
+  BrowserClient([String? cacheMode]) : _cacheMode = cacheMode ?? 'default';
```

Comment:
Why does this need to be nullable?

```suggestion
  BrowserClient({CacheMode cacheMode = CacheMode.defaultType}) : _cacheMode = cacheMode;
```